runtime/doc/eval.txt

```diff
@@ -8545,6 +8553,9 @@ sqrt({expr})						*sqrt()*
 		"nan" may be different, it depends on system libraries.
 		{only available when compiled with the |+float| feature}
 
+str2blob({expr})					*str2blob()*
```

Comment:
Perhaps there should be a 2nd optional argument {sep}, just like for `blob2str()`.
So `str2blob('ab-cd', '-')` would give `0z6162006364`.

Reply:
Added sep for str2blob